### PR TITLE
fixes #13178 - fix path to jquery assets

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

**Comment by ehelms** [comment]
ACK

File: app/assets/stylesheets/katello/katello.scss [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
 
-@import "katello/jquery-ui-1.8.11.custom.css";
-@import "katello/jquery.loadmask.css";
-@import "katello/jquery.jnotify.css";
-@import "katello/jquery.treeTable.css";
-@import "katello/jquery.jscrollpane.css";
-@import "katello/ui.spinner.css";
+@import "jquery-ui-1.8.11.custom.css";
+@import "jquery.loadmask.css";
+@import "jquery.jnotify.css";
```
Comment (ohadlevy) [inline app/assets/stylesheets/katello/katello.scss]: afair jnotify is loaded in foreman?

Comment (thomasmckay) [inline app/assets/stylesheets/katello/katello.scss]: @ohadlevy link? sample page to test that on?
@ehelms thoughts?

I won't merge until I hear back from both of you. Thanks!

Comment (ehelms) [inline app/assets/stylesheets/katello/katello.scss]: Looks roughly like the same (https://github.com/theforeman/foreman/blob/develop/vendor/assets/stylesheets/jquery.jnotify.scss) which we can remove it here and from our code base if so. I am not even sure we actually use jnotify anywhere anymore.

**Comment by ehelms** [comment]
@thomasmckay up to you if you want to do the cleanup in this PR or not, ACK either way

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
ACK

**Comment by ehelms** [comment]
Going ahead and merging this.

**Pull request merged**